phpunit/data/themedir1/block-theme-child/theme.json

```diff
@@ -1,5 +1,5 @@
 {
-	"version": 1,
+	"version": 2,
```

Comment:
I updated the version number here, which seems like it was incorrect to begin with since it was already using the `customTemplates` and `templateParts` properties which were added in v2.